pytest_selenium/pytest_selenium.py

```diff
@@ -198,6 +200,15 @@ def _gather_html(item, report, driver, summary, extra):
 
 
 def _gather_logs(item, report, driver, summary, extra):
+    pytest_html = item.config.pluginmanager.getplugin('html')
+    # Open and read geckodriver log
+    if item.config._driver_log is not None:
+        with open(item.config._driver_log, 'r') as driver_log:
+            extra.append(pytest_html.extras.text(
+                driver_log.read(), 'Geckodriver Log'))
```

Comment:
This shouldn't be implementation specific, as other drivers may also provide logs. Let's call it 'Driver Log'.